src/api-service/__app__/node/__init__.py

```diff
@@ -50,6 +50,10 @@ def post(req: func.HttpRequest) -> func.HttpResponse:
     if isinstance(request, Error):
         return not_ok(request, context="NodeUpdate")
 
+    answer = check_can_manage_pools(req)
```

Comment:
It's probably time for a more meaningful name over `check_can_manage_pools`.

Reply:
I think the underlying idea is consistent actually. This check is used for pools, scalesets, and nodes. Scalesets and nodes are just _part_ of the pool so I think it makes sense that you're still checking if _you can manage the pool_.

Reply:
The problem is that although they might be related the things that are bing handled (scalesets, nodes) _aren't_ pools, so it's miss-leading.

Reply:
If we want to change then we will need to also change the instance_config value - can_manage_pools since this is the actual value that the function is checking against.